### PR TITLE
check plugin name on plugin_reload rpc call

### PR DESCRIPTION
**Pull request by fanatid**
#### Problem

On `plugin_reload` we need to check the new plugin name, the path to the plugin can be changed in the config file, and as a result, already loaded plugin can be loaded again.

#### Summary of Changes

Add plugin name check.

### CODE REVIEW

**Comment by fanatid** [comment]
@lijunwangs can you check this, please?

File: geyser-plugin-manager/src/geyser_plugin_manager.rs [hunk 1]
```diff
@@ -177,6 +177,22 @@ impl GeyserPluginManager {
...
+        if self
+            .plugins
+            .iter()
+            .any(|plugin| plugin.name().eq(new_plugin.name()))
+        {
+            return Err(jsonrpc_core::Error {
+                code: ErrorCode::InvalidRequest,
+                message: format!(
+                    "There already exists a plugin named {} loaded. Did not load requested plugin",
+                    new_plugin.name()
```
Comment (lijunwangs) [inline geyser-plugin-manager/src/geyser_plugin_manager.rs:190]: Can you also include the input `name` in the message? Like "while reloading 'name' "

Comment (fanatid) [reply geyser-plugin-manager/src/geyser_plugin_manager.rs:190]: Done

**Review by lijunwangs — APPROVED** [review]
looks good

**Pull request merged**